numpy/core/fromnumeric.py

```diff
@@ -70,11 +70,26 @@ def take(a, indices, axis=None, out=None, mode='raise'):
     using arrays); however, it can be easier to use if you need elements
     along a given axis.
 
+    When axis is not None this is equivalent to the following use of `ndindex`,
+    which sets each of ``ii``, ``jj``, and ``kk`` to a tuple of indices:
+
+        for ii in ndindex(a.shape[:axis]):
+            for jj in ndindex(indices.shape):
+                for kk in ndindex(a.shape[axis+1:]):
+                    out[ii + jj + kk] = a[ii + (indices[jj],) + kk]
+
+    Equivalently, using `s_` to build simple slice objects, this can be
+    expressed in terms of applying fancy indexing to each 1d slice:
+
+        for ii in ndindex(a.shape[:axis]):
+            for kk in ndindex(a.shape[axis+1:]):
+                out[ii + s_[...] + kk] = a[ii + s_[:,] + kk][indices]
```

Comment:
Oops, this is inconsistent - would you prefer I use `s_[...,]` or `(...,)`? The former is required on python 2, but...